analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SillyBitwiseOperation.cs

```diff
@@ -38,7 +38,7 @@ static void Main(string[] args  )
 
             long bitMaskLong = 0x010F;
             long resultLong;
-            resultLong = bitMaskLong & - - - +1L; // Noncompliant
```

Comment:
That was not intentional and I didn't actually noticed that :( VS formatting did it.
I'd keep it like this to prevent similar PR noise in the future.